lib/ansible/utils/collection_loader/_collection_config.py

```diff
@@ -101,6 +102,10 @@ def _require_finder(cls):
             raise NotImplementedError('an AnsibleCollectionFinder has not been installed in this process')
 
 
-# concrete class of our metaclass type that defines the class properties we want
-class AnsibleCollectionConfig(metaclass=_AnsibleCollectionConfig):
-    pass
+if PY3:
```

Comment:
`six` has a `@with_metaclass` decorator IIRC so that you don't need to do this if-else dance. LMGTFY

Reply:
Oh, it's `@add_metaclass` that is a decorator and `with_metaclass()` is a function. FWIW here's the doc on both: https://six.readthedocs.io/#six.with_metaclass.

Reply:
:100: much better